bentoml/yatai/validator/deployment_pb_validator.py

```diff
@@ -119,6 +148,23 @@ def _validate_bento_service_version(self, bento_service_version, field, value):
                 'an anti-pattern.',
             )
 
+    def _validate_azure_function_min_instances_lower_than_max(
```

Comment:
it might make more sense to do a `_validate_ azure_function_operator_config` instead and simply do something like
```
value['min_instances'] < value['max_burst']
```
in the validation function?